packages/core/src/forms/control-action/control-action.element.ts

```diff
@@ -5,7 +5,7 @@
  */
 
 import { html, property } from 'lit-element';
-import { CdsIcon } from '@clr/core/icon';
+import { CdsIcon } from '@clr/core/icon/icon.element.js';
```

Comment:
This path isn't listed in importmap. Is that a concern?

Reply:
Ah good catch thanks!